src/MySqlConnector/Logging/Log.cs

```diff
@@ -47,7 +47,7 @@ internal static partial class Log
 	public static partial void AutoDetectedAurora57(ILogger logger, string sessionId, string hostName);
 
 	[LoggerMessage(EventIds.SessionMadeConnection, LogLevel.Debug, "Session {SessionId} made connection; server version {ServerVersion}; connection ID {ConnectionId}; supports: compression {SupportsCompression}, attributes {SupportsAttributes}, deprecate EOF {SupportsDeprecateEof}, cached metadata {SupportsCachedMetadata}, SSL {SupportsSsl}, session track {SupportsSessionTrack}, pipelining {SupportsPipelining}, query attributes {SupportsQueryAttributes}")]
-	public static partial void SessionMadeConnection(ILogger logger, string sessionId, string serverVersion, int connectionId, bool supportsCompression, bool supportsAttributes, bool supportsDeprecateEof, bool supportsCachedMetadata, bool supportsSsl, bool supportsSessionTrack, bool supportsPipelining, bool supportsQueryAttributes);
+	public static partial void SessionMadeConnection(ILogger logger, string sessionId, string serverVersion, long connectionId, bool supportsCompression, bool supportsAttributes, bool supportsDeprecateEof, bool supportsCachedMetadata, bool supportsSsl, bool supportsSessionTrack, bool supportsPipelining, bool supportsQueryAttributes);
```

Comment:
Was there a plan to make this variable a `long` throughout the rest of the code? Does MariaDB use 64-bit connection IDs?

https://github.com/mysql-net/MySqlConnector/pull/1494/files#diff-abcf6fc970e193538504a795eb54ea931b52864f45ef4fa0e8040ee1273dac9aR96 still converts to an Int32.

Reverted to `int` here: https://github.com/mysql-net/MySqlConnector/pull/1497/commits/b19f1ee1eed604983c87e42c6232813fbb5ecc76.

Reply:
Sorry, my fault, i began changing that and revert it, but not completly. 
technically, mariadb use internally long, but using the integer truncated value on initial handshake works ok (see https://jira.mariadb.org/browse/MDEV-15089). To be honest, i've only heard about wikipedia having an issue with that, creating more than 2147483647 connections without restart is an achievement !